roc/log_test.go

```diff
@@ -140,6 +141,47 @@ func TestLog_Func(t *testing.T) {
 	}
 }
 
+func TestLog_LogWrite(t *testing.T) {
+	SetLogLevel(LogDebug)
+	defer SetLogLevel(defaultLogLevel)
+
+	ch := make(chan LogMessage, 1)
+
+	SetLoggerFunc(func(msg LogMessage) {
+		if msg.Level == LogDebug {
+			select {
+			case ch <- msg:
+			default:
+			}
+		}
+	})
+	defer SetLoggerFunc(nil)
+
+	ctx, err := OpenContext(ContextConfig{})
+	require.NoError(t, err)
+	ctx.Close()
+
+	select {
+	case msg := <-ch:
+		assert.Equal(t, LogDebug, msg.Level, "Expected log level to be debug")
+		assert.NotEmpty(t, msg.Module)
+		assert.Equal(t, "roc_go", msg.Module)
+		assert.NotEmpty(t, msg.File)
+		assert.Equal(t, "log.go", msg.File[len(msg.File)-6:])
+		assert.NotEmpty(t, msg.Line)
+		assert.Equal(t, 192, msg.Line)
```

Comment:
Hard-coding specific line is not very good, it may frequently change due to modifications not related to this test, and it should not break the test.

Instead of NotEmpty + Equal, let's just check that line is positive number.